### ISSUE TITLE
bug: running `cargo shuttle status` outside a cargo project panics

### ISSUE THREAD

**Issue by christoshadjiaslanis** [shuttle-hq/shuttle#152]
The error message looks like this:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: CargoMetadata { stderr: "error: could not find `Cargo.toml` in `/path/to/folder` or any parent directory\n" }', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-shuttle-0.2.5/src/config.rs:224:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by AugustoFKL** [comment]
Sorry to ask, but, is there a expected behavior?

Should it be a cleaner and more reasonable error message? Or some other complex behavior?

**Comment by christoshadjiaslanis** [comment]
@AugustoFKL good point - I was in a rush when I made this. The description should now be more precise and include expected behaviour.

**Comment by AugustoFKL** [comment]
@christoshadjiaslanis that's perfect! Actually, since the behavior is stopping the application, I think an expect would be fine.

Will analyse the architecture later to see if we really need to propagate the error, and implement it.

Thanks for the update!

**Comment by christoshadjiaslanis** [comment]
@AugustoFKL - any updates on this? Need any help?

**Comment by ivancernja** [comment]
@christoshadjiaslanis I'll have this fixed under this PR: https://github.com/shuttle-hq/shuttle/pull/203 if you and @AugustoFKL don't mind.
